compiler/src/iree/compiler/Dialect/HAL/Target/CUDA/CUDATarget.cpp

```diff
@@ -251,108 +291,142 @@ class CUDATargetBackend final : public TargetBackend {
 
     SmallVector<std::string> entryPointNames;
     std::string ptxImage;
-    if (variantOp.isExternal()) {
```

Comment:
All this changes are actually just space differences (because I increased the indentation, I didn't want to refactor anymore code yet).
With a `diff -b` the changes are much more obvious.